diff --git a/node_modules/@react-native-community/datetimepicker/android/src/main/java/com/reactcommunity/rndatetimepicker/MaterialDatePickerModule.kt b/node_modules/@react-native-community/datetimepicker/android/src/main/java/com/reactcommunity/rndatetimepicker/MaterialDatePickerModule.kt
index 2cb28ad..6f5a3f5 100644
--- a/node_modules/@react-native-community/datetimepicker/android/src/main/java/com/reactcommunity/rndatetimepicker/MaterialDatePickerModule.kt
+++ b/node_modules/@react-native-community/datetimepicker/android/src/main/java/com/reactcommunity/rndatetimepicker/MaterialDatePickerModule.kt
@@ -18,12 +18,12 @@ class MaterialDatePickerModule(reactContext: ReactApplicationContext): NativeMod
   }
 
   override fun dismiss(promise: Promise?) {
-    val activity = currentActivity as FragmentActivity?
+    val activity = getCurrentActivity() as FragmentActivity?
     dismissDialog(activity, NAME, promise)
   }
 
   override fun open(params: ReadableMap, promise: Promise) {
-    val activity = currentActivity as FragmentActivity?
+    val activity = getCurrentActivity() as FragmentActivity?
     if (activity == null) {
       promise.reject(
         RNConstants.ERROR_NO_ACTIVITY,
diff --git a/node_modules/@react-native-community/datetimepicker/android/src/main/java/com/reactcommunity/rndatetimepicker/MaterialTimePickerModule.kt b/node_modules/@react-native-community/datetimepicker/android/src/main/java/com/reactcommunity/rndatetimepicker/MaterialTimePickerModule.kt
index b3e7566..bcae232 100644
--- a/node_modules/@react-native-community/datetimepicker/android/src/main/java/com/reactcommunity/rndatetimepicker/MaterialTimePickerModule.kt
+++ b/node_modules/@react-native-community/datetimepicker/android/src/main/java/com/reactcommunity/rndatetimepicker/MaterialTimePickerModule.kt
@@ -19,12 +19,12 @@ class MaterialTimePickerModule(reactContext: ReactApplicationContext) :
   }
 
   override fun dismiss(promise: Promise?) {
-    val activity = currentActivity as FragmentActivity?
+    val activity = getCurrentActivity() as FragmentActivity?
     dismissDialog(activity, NAME, promise)
   }
 
   override fun open(params: ReadableMap, promise: Promise) {
-    val activity = currentActivity as FragmentActivity?
+    val activity = getCurrentActivity() as FragmentActivity?
     if (activity == null) {
       promise.reject(
         RNConstants.ERROR_NO_ACTIVITY,
