diff --git a/node_modules/edge-login-ui-rn/src/components/common/FormField.js b/node_modules/edge-login-ui-rn/src/components/common/FormField.js
index 30bc17b..3640849 100644
--- a/node_modules/edge-login-ui-rn/src/components/common/FormField.js
+++ b/node_modules/edge-login-ui-rn/src/components/common/FormField.js
@@ -12,10 +12,10 @@ type Props = {
   autoCorrect: boolean,
   autoFocus: boolean,
   forceFocus: boolean,
-  autoCapitalize?: string,
+  autoCapitalize?: 'none' | 'sentences' | 'words' | 'characters',
   secureTextEntry: boolean,
   showSecureCheckbox: boolean,
-  returnKeyType: string,
+  returnKeyType?: 'done' | 'go' | 'next' | 'search' | 'send',
   error?: string,
   onFinish(): void,
   onFocus(): void,
diff --git a/node_modules/edge-login-ui-rn/src/components/common/MaterialInput.js b/node_modules/edge-login-ui-rn/src/components/common/MaterialInput.js
index 8948356..cf56e7b 100644
--- a/node_modules/edge-login-ui-rn/src/components/common/MaterialInput.js
+++ b/node_modules/edge-login-ui-rn/src/components/common/MaterialInput.js
@@ -16,11 +16,11 @@ type Props = {
   errorColor: string,
   titleTextStyle: Object,
   secureTextEntry: boolean,
-  autoCapitalize?: string,
+  autoCapitalize?: 'none' | 'sentences' | 'words' | 'characters',
   autoCorrect: boolean,
   autoFocus: boolean,
   forceFocus: boolean,
-  returnKeyType: string,
+  returnKeyType?: 'done' | 'go' | 'next' | 'search' | 'send',
   onFocus(): void,
   onBlur(): void,
   onChangeText(string): void,
@@ -107,7 +107,7 @@ export class MaterialInput extends React.Component<Props, State> {
     )
   }

-  addRef = (arg: TextField) => {
+  addRef = (arg: TextField | null) => {
     if (arg) {
       this.textInput = arg
     }
