diff --git a/node_modules/edge-components/src/Modals/SimpleConfirmModal/SimpleConfirmModal.js b/node_modules/edge-components/src/Modals/SimpleConfirmModal/SimpleConfirmModal.js
index 40112b8..0e056d2 100644
--- a/node_modules/edge-components/src/Modals/SimpleConfirmModal/SimpleConfirmModal.js
+++ b/node_modules/edge-components/src/Modals/SimpleConfirmModal/SimpleConfirmModal.js
@@ -191,7 +191,7 @@ export class SimpleConfirmModal extends Component<Props> {
     const { isActive, style, ...props } = this.props
     const children = React.Children.toArray(this.props.children)
     const icon = children.find(child => child.type === SimpleConfirmModal.Icon)
-    const title = children.find(child => child.type === SimpleConfirmModal.Title)
+    const title = children.find(child => child.type === SimpleConfirmModal.Title) || null
     const body = children.find(child => child.type === SimpleConfirmModal.Body)
     const footer = children.find(child => child.type === SimpleConfirmModal.Footer)
 
diff --git a/node_modules/edge-components/src/Modals/ThreeButtonModal/ThreeButtonModal.js b/node_modules/edge-components/src/Modals/ThreeButtonModal/ThreeButtonModal.js
index bc9f66c..bd1c4e1 100644
--- a/node_modules/edge-components/src/Modals/ThreeButtonModal/ThreeButtonModal.js
+++ b/node_modules/edge-components/src/Modals/ThreeButtonModal/ThreeButtonModal.js
@@ -191,7 +191,7 @@ export class ThreeButtonModal extends Component<Props> {
     const { isActive, style, ...props } = this.props
     const children = React.Children.toArray(this.props.children)
     const icon = children.find(child => child.type === ThreeButtonModal.Icon)
-    const title = children.find(child => child.type === ThreeButtonModal.Title)
+    const title = children.find(child => child.type === ThreeButtonModal.Title) || null
     const body = children.find(child => child.type === ThreeButtonModal.Body)
     const footer = children.find(child => child.type === ThreeButtonModal.Footer)
 
diff --git a/node_modules/edge-components/src/Modals/YesNoModal/YesNoModal.js b/node_modules/edge-components/src/Modals/YesNoModal/YesNoModal.js
index 8576825..e3b1104 100644
--- a/node_modules/edge-components/src/Modals/YesNoModal/YesNoModal.js
+++ b/node_modules/edge-components/src/Modals/YesNoModal/YesNoModal.js
@@ -191,7 +191,7 @@ export class YesNoModal extends Component<Props> {
     const { isActive, style, ...props } = this.props
     const children = React.Children.toArray(this.props.children)
     const icon = children.find(child => child.type === YesNoModal.Icon)
-    const title = children.find(child => child.type === YesNoModal.Title)
+    const title = children.find(child => child.type === YesNoModal.Title) || null
     const body = children.find(child => child.type === YesNoModal.Body)
     const footer = children.find(child => child.type === YesNoModal.Footer)
 
