diff --git a/node_modules/expo-file-system/ios/FileSystemModule.swift b/node_modules/expo-file-system/ios/FileSystemModule.swift
index 3519c67..33ea82a 100644
--- a/node_modules/expo-file-system/ios/FileSystemModule.swift
+++ b/node_modules/expo-file-system/ios/FileSystemModule.swift
@@ -256,23 +256,11 @@ public final class FileSystemModule: Module {
     }
 
     AsyncFunction("getFreeDiskStorageAsync") { () -> Int in
-    // Uses required reason API based on the following reason: E174.1 85F4.1
-      let resourceValues = try getResourceValues(from: documentDirectory, forKeys: [.volumeAvailableCapacityKey])
-
-      guard let availableCapacity = resourceValues?.volumeAvailableCapacity else {
-        throw CannotDetermineDiskCapacity()
-      }
-      return availableCapacity
+      return 0
     }
 
     AsyncFunction("getTotalDiskCapacityAsync") { () -> Int in
-        // Uses required reason API based on the following reason: E174.1 85F4.1
-      let resourceValues = try getResourceValues(from: documentDirectory, forKeys: [.volumeTotalCapacityKey])
-
-      guard let totalCapacity = resourceValues?.volumeTotalCapacity else {
-        throw CannotDetermineDiskCapacity()
-      }
-      return totalCapacity
+      return 0
     }
   }
 }
