diff --git a/node_modules/react-native-reanimated/src/Colors.ts b/node_modules/react-native-reanimated/src/Colors.ts
index cd577a6..88f01e7 100644
--- a/node_modules/react-native-reanimated/src/Colors.ts
+++ b/node_modules/react-native-reanimated/src/Colors.ts
@@ -521,7 +521,7 @@ export const rgbaColor = (
   alpha = 1
 ): number | string => {
   'worklet';
-  if (IS_WEB || !_WORKLET) {
+  if (!IS_ANDROID || !_WORKLET) {
     // Replace tiny values like 1.234e-11 with 0:
     const safeAlpha = alpha < 0.001 ? 0 : alpha;
     return `rgba(${r}, ${g}, ${b}, ${safeAlpha})`;
