diff --git a/node_modules/react-native-smart-splash-screen/android/build.gradle b/node_modules/react-native-smart-splash-screen/android/build.gradle
index dfbd150..16b8fdb 100644
--- a/node_modules/react-native-smart-splash-screen/android/build.gradle
+++ b/node_modules/react-native-smart-splash-screen/android/build.gradle
@@ -1,12 +1,16 @@
 apply plugin: 'com.android.library'
 
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.3"
+    compileSdkVersion safeExtGet('compileSdkVersion', 23)
+    buildToolsVersion safeExtGet('buildToolsVersion', '23.0.3')
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 23
+        minSdkVersion safeExtGet('minSdkVersion', 16)
+        targetSdkVersion safeExtGet('targetSdkVersion', 23)
         versionCode 1
         versionName "1.0"
     }
@@ -19,7 +23,6 @@ android {
 }
 
 dependencies {
-    compile fileTree(include: ['*.jar'], dir: 'libs')
-    compile 'com.android.support:appcompat-v7:23.4.0'
-    compile 'com.facebook.react:react-native:+'
+    implementation 'com.facebook.react:react-native:+'
+    implementation "androidx.appcompat:appcompat:1.4.1"
 }
diff --git a/node_modules/react-native-smart-splash-screen/android/src/main/java/com/reactnativecomponent/splashscreen/RCTSplashScreenModule.java b/node_modules/react-native-smart-splash-screen/android/src/main/java/com/reactnativecomponent/splashscreen/RCTSplashScreenModule.java
index 9679439..34b577a 100644
--- a/node_modules/react-native-smart-splash-screen/android/src/main/java/com/reactnativecomponent/splashscreen/RCTSplashScreenModule.java
+++ b/node_modules/react-native-smart-splash-screen/android/src/main/java/com/reactnativecomponent/splashscreen/RCTSplashScreenModule.java
@@ -1,7 +1,7 @@
 package com.reactnativecomponent.splashscreen;
 
 import android.os.Handler;
-import android.support.annotation.Nullable;
+import androidx.annotation.Nullable;
 
 import com.facebook.react.bridge.ReactApplicationContext;
 import com.facebook.react.bridge.ReactContextBaseJavaModule;
diff --git a/node_modules/react-native-smart-splash-screen/ios/RCTSplashScreen/RCTSplashScreen/RCTSplashScreen.m b/node_modules/react-native-smart-splash-screen/ios/RCTSplashScreen/RCTSplashScreen/RCTSplashScreen.m
index 278fefd..8868489 100644
--- a/node_modules/react-native-smart-splash-screen/ios/RCTSplashScreen/RCTSplashScreen/RCTSplashScreen.m
+++ b/node_modules/react-native-smart-splash-screen/ios/RCTSplashScreen/RCTSplashScreen/RCTSplashScreen.m
@@ -11,6 +11,10 @@ @implementation RCTSplashScreen
 
 RCT_EXPORT_MODULE(SplashScreen)
 
++ (BOOL)requiresMainQueueSetup
+{
+  return YES;
+}
 
 + (void)open:(RCTRootView *)v {
     [RCTSplashScreen open:v withImageNamed:@"splash"];
diff --git a/node_modules/react-native-smart-splash-screen/react-native-smart-splash-screen.podspec b/node_modules/react-native-smart-splash-screen/react-native-smart-splash-screen.podspec
new file mode 100644
index 0000000..b72b297
--- /dev/null
+++ b/node_modules/react-native-smart-splash-screen/react-native-smart-splash-screen.podspec
@@ -0,0 +1,17 @@
+require "json"
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+
+Pod::Spec.new do |s|
+  s.name         = package['name']
+  s.version      = package['version']
+  s.summary      = 'todo'
+  s.homepage     = 'todo'
+  s.license      = 'todo'
+  s.authors      = 'todo'
+  s.platform     = :ios, "8.0"
+  s.requires_arc = true
+  s.source       = { :git => "https://github.com/EdgeApp/edge-react-gui.git" }
+  s.source_files = "ios/**/*.{h,m}"
+
+  s.dependency "React"
+end
