diff --git a/node_modules/react-native-sortable-listview/index.js b/node_modules/react-native-sortable-listview/index.js
index e5a97e7..5fb5e0e 100644
--- a/node_modules/react-native-sortable-listview/index.js
+++ b/node_modules/react-native-sortable-listview/index.js
@@ -458,9 +458,9 @@ class SortableListView extends React.Component {
   }
 
   componentDidMount() {
-    InteractionManager.runAfterInteractions(() => {
+    // InteractionManager.runAfterInteractions(() => {
       this.timer = setTimeout(() => this && this.measureWrapper(), 0)
-    })
+    // })
   }
 
   componentWillReceiveProps(props) {
