diff --git a/node_modules/react-native-bootsplash/react-native.config.js b/node_modules/react-native-bootsplash/react-native.config.js
index 2f63b22..e465992 100644
--- a/node_modules/react-native-bootsplash/react-native.config.js
+++ b/node_modules/react-native-bootsplash/react-native.config.js
@@ -1,5 +1,4 @@
 const path = require("path");
-const { generate } = require("./dist/commonjs/generate");
 
 module.exports = {
   commands: [
@@ -45,6 +44,7 @@ module.exports = {
         { project: { android, ios } },
         { backgroundColor, logoWidth, assetsPath, flavor, platforms },
       ) => {
+        const { generate } = require("./dist/commonjs/generate");
         const workingPath =
           process.env.INIT_CWD || process.env.PWD || process.cwd();
 
