diff --git a/node_modules/react-native-smart-splash-screen/android/build.gradle b/node_modules/react-native-smart-splash-screen/android/build.gradle
index dfbd150..6980b86 100644
--- a/node_modules/react-native-smart-splash-screen/android/build.gradle
+++ b/node_modules/react-native-smart-splash-screen/android/build.gradle
@@ -1,12 +1,16 @@
 apply plugin: 'com.android.library'
 
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.3"
+    compileSdkVersion safeExtGet('compileSdkVersion', 23)
+    buildToolsVersion safeExtGet('buildToolsVersion', '23.0.3')
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 23
+        minSdkVersion safeExtGet('minSdkVersion', 16)
+        targetSdkVersion safeExtGet('targetSdkVersion', 23)
         versionCode 1
         versionName "1.0"
     }
@@ -20,6 +24,6 @@ android {
 
 dependencies {
     compile fileTree(include: ['*.jar'], dir: 'libs')
-    compile 'com.android.support:appcompat-v7:23.4.0'
+    compile 'com.android.support:appcompat-v7:${safeExtGet("supportLibVersion", "23.4.0")}'
     compile 'com.facebook.react:react-native:+'
 }
diff --git a/node_modules/react-native-smart-splash-screen/react-native-smart-splash-screen.podspec b/node_modules/react-native-smart-splash-screen/react-native-smart-splash-screen.podspec
new file mode 100644
index 0000000..b72b297
--- /dev/null
+++ b/node_modules/react-native-smart-splash-screen/react-native-smart-splash-screen.podspec
@@ -0,0 +1,17 @@
+require "json"
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+
+Pod::Spec.new do |s|
+  s.name         = package['name']
+  s.version      = package['version']
+  s.summary      = 'todo'
+  s.homepage     = 'todo'
+  s.license      = 'todo'
+  s.authors      = 'todo'
+  s.platform     = :ios, "8.0"
+  s.requires_arc = true
+  s.source       = { :git => "https://github.com/EdgeApp/edge-react-gui.git" }
+  s.source_files = "ios/**/*.{h,m}"
+
+  s.dependency "React"
+end
