diff --git a/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java b/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java
index 0a57f4f..0df0214 100644
--- a/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java
+++ b/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java
@@ -945,6 +945,7 @@ public class RNCWebViewManager extends SimpleViewManager<WebView> {
       }
       WritableMap event = Arguments.createMap();
       event.putDouble("target", webView.getId());
+      event.putString("url", webView.getUrl());
       event.putString("title", webView.getTitle());
       event.putString("url", url);
       event.putBoolean("canGoBack", webView.canGoBack());
diff --git a/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewModule.java b/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewModule.java
index 5b5ffe2..622ea0a 100644
--- a/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewModule.java
+++ b/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewModule.java
@@ -272,11 +272,6 @@ public class RNCWebViewModule extends ReactContextBaseJavaModule implements Acti
   }

   public boolean grantFileDownloaderPermissions() {
-    // Permission not required for Android Q and above
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
-      return true;
-    }
-
     boolean result = ContextCompat.checkSelfPermission(getCurrentActivity(), Manifest.permission.WRITE_EXTERNAL_STORAGE) == PackageManager.PERMISSION_GRANTED;
     if (!result && Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
       PermissionAwareActivity activity = getPermissionAwareActivity();
diff --git a/node_modules/react-native-webview/lib/WebView.android.js b/node_modules/react-native-webview/lib/WebView.android.js
index 6d60e27..1db3c9e 100644
--- a/node_modules/react-native-webview/lib/WebView.android.js
+++ b/node_modules/react-native-webview/lib/WebView.android.js
@@ -211,6 +211,8 @@ var WebView = /** @class */ (function (_super) {
             if (onLoadProgress) {
                 onLoadProgress(event);
             }
+            // @ts-ignore
+            _this.updateNavigationState(event);
         };
         _this.onShouldStartLoadWithRequestCallback = function (shouldStart, url) {
             if (shouldStart) {
diff --git a/node_modules/react-native-webview/lib/WebViewShared.js b/node_modules/react-native-webview/lib/WebViewShared.js
index 893d48e..f6b51a1 100644
--- a/node_modules/react-native-webview/lib/WebViewShared.js
+++ b/node_modules/react-native-webview/lib/WebViewShared.js
@@ -15,7 +15,7 @@ var extractOrigin = function (url) {
     return result === null ? '' : result[0];
 };
 var originWhitelistToRegex = function (originWhitelist) {
-    return "^" + escapeStringRegexp(originWhitelist).replace(/\\\*/g, '.*');
+    return "^" + escapeStringRegexp(originWhitelist).replace(/\\\*/g, '.*') + "$";
 };
 var passesWhitelist = function (compiledWhitelist, url) {
     var origin = extractOrigin(url);
