diff --git a/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributeProps.java b/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributeProps.java
index ae8bb00..a84c31b 100644
--- a/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributeProps.java
+++ b/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributeProps.java
@@ -397,8 +397,7 @@ public class TextAttributeProps {
             : PixelUtil.toPixelFromDIP(mLetterSpacingInput);
 
     if (mFontSize <= 0) {
-      throw new IllegalArgumentException(
-          "FontSize should be a positive value. Current value: " + mFontSize);
+      return 0;
     }
     // `letterSpacingPixels` and `mFontSize` are both in pixels,
     // yielding an accurate em value.
