diff --git a/node_modules/react-native-fast-crypto/android/build.gradle b/node_modules/react-native-fast-crypto/android/build.gradle
index dd999fa..227daac 100644
--- a/node_modules/react-native-fast-crypto/android/build.gradle
+++ b/node_modules/react-native-fast-crypto/android/build.gradle
@@ -27,7 +27,7 @@ android {
 
         externalNativeBuild {
             cmake {
-                arguments '-DANDROID_TOOLCHAIN=clang', '-DANDROID_STL=c++_shared'
+                arguments '-DANDROID_TOOLCHAIN=clang'
             }
         }
 
@@ -47,15 +47,6 @@ android {
             path "src/main/cpp/CMakeLists.txt"
         }
     }
-
-    // If you want Gradle to package prebuilt native libraries
-    // with your APK, modify the default source set configuration
-    // to include the directory of your prebuilt .so files as follows.
-    sourceSets {
-        main {
-            jniLibs.srcDirs 'jni/libs/'
-        }
-    }
 }
 
 repositories {
