diff --git a/node_modules/react-native-gifted-charts/dist/Components/common/StripAndLabel.js b/node_modules/react-native-gifted-charts/dist/Components/common/StripAndLabel.js
index fa54a00..8d98797 100644
--- a/node_modules/react-native-gifted-charts/dist/Components/common/StripAndLabel.js
+++ b/node_modules/react-native-gifted-charts/dist/Components/common/StripAndLabel.js
@@ -16,9 +16,10 @@ export var StripAndLabel = function (props) {
                 : showPointerStrip) ? (_jsxs(View, { style: {
                     position: 'absolute',
                     left: -pointerStripWidth / 4,
-                    top: containsNegative ? 0 : -pointerYLocal + 8 + xAxisThickness,
+                    top: containsNegative ? 0 : -pointerYLocal + xAxisThickness - 3,
+                    bottom: 0,
                     width: width,
-                    height: containerHeight,
+                    height: containerHeight + xAxisThickness + 1,
                 }, children: [_jsxs(Svg, { children: [_jsx(Line, { stroke: pointerStripColor, strokeWidth: pointerStripWidth, strokeDasharray: (pointerConfig === null || pointerConfig === void 0 ? void 0 : pointerConfig.strokeDashArray)
                                     ? pointerConfig === null || pointerConfig === void 0 ? void 0 : pointerConfig.strokeDashArray
                                     : '', x1: pointerX +
@@ -26,12 +27,12 @@ export var StripAndLabel = function (props) {
                                     2 -
                                     pointerStripWidth / 2 +
                                     (((_a = pointerItemLocal[0]) === null || _a === void 0 ? void 0 : _a.pointerShiftX) || 0), y1: pointerStripUptoDataPoint
-                                    ? pointerYLocal + pointerRadius - 4
+                                    ? pointerYLocal
                                     : containerHeight - pointerStripHeight, x2: pointerX +
                                     pointerRadius +
                                     2 -
                                     pointerStripWidth / 2 +
-                                    (((_b = pointerItemLocal[0]) === null || _b === void 0 ? void 0 : _b.pointerShiftX) || 0), y2: containerHeight }), horizontalStripConfig && (_jsx(Line, { stroke: (_c = horizontalStripConfig.color) !== null && _c !== void 0 ? _c : pointerStripColor, strokeWidth: (_d = horizontalStripConfig.thickness) !== null && _d !== void 0 ? _d : pointerStripWidth, strokeDasharray: ((_f = (_e = pointerConfig === null || pointerConfig === void 0 ? void 0 : pointerConfig.horizontalStripConfig) === null || _e === void 0 ? void 0 : _e.strokeDashArray) !== null && _f !== void 0 ? _f : pointerConfig === null || pointerConfig === void 0 ? void 0 : pointerConfig.strokeDashArray)
+                                    (((_b = pointerItemLocal[0]) === null || _b === void 0 ? void 0 : _b.pointerShiftX) || 0), y2: containerHeight + xAxisThickness + 1 }), horizontalStripConfig && (_jsx(Line, { stroke: (_c = horizontalStripConfig.color) !== null && _c !== void 0 ? _c : pointerStripColor, strokeWidth: (_d = horizontalStripConfig.thickness) !== null && _d !== void 0 ? _d : pointerStripWidth, strokeDasharray: ((_f = (_e = pointerConfig === null || pointerConfig === void 0 ? void 0 : pointerConfig.horizontalStripConfig) === null || _e === void 0 ? void 0 : _e.strokeDashArray) !== null && _f !== void 0 ? _f : pointerConfig === null || pointerConfig === void 0 ? void 0 : pointerConfig.strokeDashArray)
                                     ? pointerConfig === null || pointerConfig === void 0 ? void 0 : pointerConfig.strokeDashArray
                                     : '', x1: 0, y1: pointerYLocal - 7, x2: horizontalStripConfig.horizontalStripUptoDataPoint
                                     ? pointerX + 2
