diff --git a/node_modules/react-native-tcp/android/gradle.properties b/node_modules/react-native-tcp/android/gradle.properties
deleted file mode 100644
index d015431..0000000
--- a/node_modules/react-native-tcp/android/gradle.properties
+++ /dev/null
@@ -1,2 +0,0 @@
-android.useAndroidX=true
-android.enableJetifier=true
\ No newline at end of file
diff --git a/node_modules/react-native-tcp/android/src/main/java/com/peel/react/TcpSockets.java b/node_modules/react-native-tcp/android/src/main/java/com/peel/react/TcpSockets.java
index 8e6d5d5..330fac1 100644
--- a/node_modules/react-native-tcp/android/src/main/java/com/peel/react/TcpSockets.java
+++ b/node_modules/react-native-tcp/android/src/main/java/com/peel/react/TcpSockets.java
@@ -83,6 +83,14 @@ public final class TcpSockets extends ReactContextBaseJavaModule implements TcpS
             .emit(eventName, params);
     }
 
+    @ReactMethod
+    public void addListener(String eventName) {
+    }
+
+    @ReactMethod
+    public void removeListeners(Integer count) {
+    }
+
     @ReactMethod
     public void listen(final Integer cId, final String host, final Integer port) {
         new GuardedAsyncTask<Void, Void>(getReactApplicationContext()) {
