diff --git a/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewModuleImpl.java b/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewModuleImpl.java
index d5f3533..b4cc669 100644
--- a/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewModuleImpl.java
+++ b/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewModuleImpl.java
@@ -325,10 +325,6 @@ public class RNCWebViewModuleImpl implements ActivityEventListener {
 
     public boolean grantFileDownloaderPermissions(String downloadingMessage, String lackPermissionToDownloadMessage) {
         Activity activity = mContext.getCurrentActivity();
-        // Permission not required for Android Q and above
-        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.P) {
-            return true;
-        }
 
         boolean result = ContextCompat.checkSelfPermission(activity, Manifest.permission.WRITE_EXTERNAL_STORAGE) == PackageManager.PERMISSION_GRANTED;
         if (!result && Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
diff --git a/node_modules/react-native-webview/src/WebViewShared.tsx b/node_modules/react-native-webview/src/WebViewShared.tsx
index 08905ea..2d8b360 100644
--- a/node_modules/react-native-webview/src/WebViewShared.tsx
+++ b/node_modules/react-native-webview/src/WebViewShared.tsx
@@ -25,7 +25,7 @@ const extractOrigin = (url: string): string => {
 };
 
 const originWhitelistToRegex = (originWhitelist: string): string =>
-  `^${escapeStringRegexp(originWhitelist).replace(/\\\*/g, '.*')}`;
+  `^${escapeStringRegexp(originWhitelist).replace(/\\\*/g, '.*')}$`;
 
 const passesWhitelist = (compiledWhitelist: readonly string[], url: string) => {
   const origin = extractOrigin(url);
@@ -240,6 +240,8 @@ export const useWebViewLogic = ({
       }
       // !patch for Android only
       onLoadProgress?.(event);
+      // @ts-expect-error
+      updateNavigationState(event);
     },
     [onLoadProgress]
   );
