diff --git a/node_modules/react-native-securerandom/android/build.gradle b/node_modules/react-native-securerandom/android/build.gradle
index 4e725c3..b92e86a 100644
--- a/node_modules/react-native-securerandom/android/build.gradle
+++ b/node_modules/react-native-securerandom/android/build.gradle
@@ -6,7 +6,6 @@ buildscript {
         repositories {
             google()
             mavenCentral()
-            jcenter()
         }
 
         dependencies {
@@ -56,7 +55,6 @@ repositories {
                 excludeGroup "com.facebook.react"
             }
         }
-    jcenter()
 }
 
 dependencies {
