diff --git a/node_modules/rn-qr-generator/android/build.gradle b/node_modules/rn-qr-generator/android/build.gradle
index 0f6b87c..a473742 100644
--- a/node_modules/rn-qr-generator/android/build.gradle
+++ b/node_modules/rn-qr-generator/android/build.gradle
@@ -4,7 +4,8 @@ def safeExtGet(prop, fallback) {
 
 buildscript {
     repositories {
-        jcenter()
+        google()
+        mavenCentral()
     }
 
     dependencies {
