diff --git a/android/app/build.gradle b/android/app/build.gradle
index 83029d7e7..655643c8d 100644
--- a/android/app/build.gradle
+++ b/android/app/build.gradle
@@ -100,7 +100,7 @@ android {
 
     namespace "co.edgesecure.app"
     defaultConfig {
-        applicationId "co.edgesecure.app"
+        applicationId "app.edge.develop"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode 21000000
