diff --git a/ios/SplashScreenResource/LaunchScreen.xib b/ios/SplashScreenResource/LaunchScreen.xib
index 044317a74..ba470c847 100644
--- a/ios/SplashScreenResource/LaunchScreen.xib
+++ b/ios/SplashScreenResource/LaunchScreen.xib
@@ -13,7 +13,7 @@
             <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
             <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
             <subviews>
-                <imageView userInteractionEnabled="NO" contentMode="scaleAspectFill" image="splash.png" translatesAutoresizingMaskIntoConstraints="NO" id="ZkI-RL-69Z">
+                <imageView userInteractionEnabled="NO" contentMode="scaleAspectFill" image="splash-coinhub.png" translatesAutoresizingMaskIntoConstraints="NO" id="ZkI-RL-69Z">
                     <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
                 </imageView>
             </subviews>
@@ -28,6 +28,6 @@
         <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
     </objects>
     <resources>
-        <image name="splash.png" width="750" height="1334"/>
+        <image name="splash-coinhub.png" width="750" height="1334"/>
     </resources>
 </document>
