diff --git a/android/app/build.gradle b/android/app/build.gradle
index 83029d7e7..e565d89cf 100644
--- a/android/app/build.gradle
+++ b/android/app/build.gradle
@@ -98,9 +98,9 @@ android {
 
     compileSdkVersion rootProject.ext.compileSdkVersion
 
-    namespace "co.edgesecure.app"
+    namespace "app.edge.beta"
     defaultConfig {
-        applicationId "co.edgesecure.app"
+        applicationId "app.edge.beta"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode 21000000
